Dart/src/com/jetbrains/lang/dart/ide/formatter/settings/DartLanguageCodeStyleSettingsProvider.java

```diff
@@ -50,8 +50,12 @@ public CommonCodeStyleSettings getDefaultCommonSettings() {
     defaultSettings.BINARY_OPERATION_WRAP = CommonCodeStyleSettings.WRAP_AS_NEEDED;
     defaultSettings.ASSIGNMENT_WRAP = CommonCodeStyleSettings.WRAP_AS_NEEDED; // TODO Remove if not needed.
     defaultSettings.ARRAY_INITIALIZER_WRAP = CommonCodeStyleSettings.WRAP_AS_NEEDED;
-    defaultSettings.TERNARY_OPERATION_WRAP = CommonCodeStyleSettings.WRAP_AS_NEEDED;
+    defaultSettings.TERNARY_OPERATION_WRAP = CommonCodeStyleSettings.WRAP_ON_EVERY_ITEM;
```

Comment:
WRAP_ON_EVERY_ITEM is not a valid value for this option. You get million of NPEs when you open Wrapping and Braced tab of the Dart settings. Did you notice them in debug console?

I spent half an hour hunting for the answer. And here it is: see the code where [TERNARY_OPERATION_WRAP](https://github.com/JetBrains/intellij-community/blob/9eb2ba1e8ce1d0b47e8f12f7c403712cb9039e56/platform/lang-impl/src/com/intellij/application/options/codeStyle/WrappingAndBracesPanel.java#L146) option is declared.
[CodeStyleSettingsCustomizable#WRAP_VALUES](https://github.com/JetBrains/intellij-community/blob/9eb2ba1e8ce1d0b47e8f12f7c403712cb9039e56/platform/lang-api/src/com/intellij/psi/codeStyle/CodeStyleSettingsCustomizable.java#L256) array contains possible values of TERNARY_OPERATION_WRAP option. And there is no WRAP_ON_EVERY_ITEM, but instead there is WRAP_AS_NEEDED | WRAP_ON_EVERY_ITEM which is equal to 5.